src/js/components/AppDebugInfoComponent.jsx

```diff
@@ -56,9 +57,13 @@ var AppDebugInfoComponent = React.createClass({
         <dt>Host</dt>
         <dd>{lastTaskFailure.host}</dd>
         <dt>Timestamp</dt>
-        <dd>{lastTaskFailure.timestamp}</dd>
+        <dd>
+          <span>{lastTaskFailure.timestamp}</span> ({new Moment(lastTaskFailure.timestamp).fromNow()})
```

Comment:
Yes, but I am not sure if this is more readable/expressive, because then the ```var timestampRelative``` must be created above the markup, what destroys the relationship.

And I am sorry, have to do a little bit nitpicking. :)
This is not logic, because here is no condition.